tests/appveyor/mssql.sql2014.pdo_sqlsrv.appveyor.xml

```diff
@@ -13,11 +13,13 @@
         <var name="db_username" value="sa" />
         <var name="db_password" value="Password12!" />
         <var name="db_name" value="doctrine_tests" />
+        <var name="db_port" value="1433"/>
 
         <var name="tmpdb_type" value="pdo_sqlsrv"/>
         <var name="tmpdb_host" value="(local)\SQL2014" />
         <var name="tmpdb_username" value="sa" />
         <var name="tmpdb_password" value="Password12!" />
+        <var name="db_port" value="1433"/>
```

Comment:
Oops, I overlooked that when I was trying to edit it on my phone. Thanks for making the adjustment.